### PR TITLE
use secure cookies in production, closes #95

### LINKED ISSUE #95: Enable Secure cookies

### PR DESCRIPTION
**Pull request by orischwartz**

### CODE REVIEW

**Comment by orischwartz** [comment]
See also #69 -- we disable the global `force_ssl` configuration option for the ELB health check.

File: config/initializers/session_store.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Be sure to restart your server when you modify this file.
 
-Rails.application.config.session_store :cookie_store, key: '_efolder-express_session', expire_after: 2.weeks
+Rails.application.config.session_store :cookie_store, key: '_efolder-express_session', expire_after: 2.weeks, secure: Rails.env.production?
```
Comment (paultag) [inline config/initializers/session_store.rb]: Will this break the healthcheck endpoint?

Comment (orischwartz) [inline config/initializers/session_store.rb]: No. This only affects the ability to set a cookie in the response, unlike the `force_ssl` config option which enforces both secure cookies and forces an http->https redirect on all requests.

**Comment by orischwartz** [comment]
I'm going to merge this myself -- per @gmorgana-va we need this today.

**Pull request merged**